packages/storybook-utils/src/preview.ts

```diff
@@ -40,7 +35,15 @@ const themes = {
  * export default preview;
  * ```
  */
-export const createPreview = <T extends Preview = Preview>(overrides?: T) => {
+export const createPreview = <T extends Preview = Preview>(
+  brandDetails: BrandDetails,
+  overrides?: T,
```

Comment:
Would prefer to make the brandDetails optional as second parameter since most projects probably don't care about this

Reply:
No, because if anybody uses it should not default to our details.
This will create the impression that it is an official `onyx` product.

Reply:
I meant to not provide any defaults but make the parameter optional.
If not provided, it will fall back to the default Storybook logo. We shouldn't force projects to pass any brand details, there might not be any

Reply:
done